### PR TITLE
Finish flask restx migration

### LINKED ISSUE #2809: Migrate API to flask-restx
Follow up from #1865

Migrate API_v3 to flask-restx and create documentation for it (models, parsers, docs, ...)

API endpoint namespaces TODO list:
- [x] #3000
- [ ] #2997
- [x] #2999
- [x] mock chroots (#2998)
- [x] modules (#2998)
- [x] monitor (#2998)
- [x] packages (#2996)
- [x] #3001
- [x] project chroots (#2996)
- [x] #2995
- [x] #3002
- [x] webhooks (#2998)

Other follow-ups:
  - [ ] #3029
  - [ ] #3031

### LINKED ISSUE #2997: Migrate api_v3/builds to flask-restx
Migrate api_v3/builds namespace to flask-restx, included documentation, models and parsers. The GET /build endpoint is done.

### PR DESCRIPTION
**Pull request by nikromen**
review this after #3168

Fixes #2809, fixes #2997

#2809 has some optional issues but the main migration is done so I think we can close the epic and keep those optional as individuals

### CODE REVIEW

**Review by FrostyX — COMMENTED** [review]
Otherwise LGTM, thank you:-)

**Comment by FrostyX** [comment]
The only shameful skeleton in the closet that remains is the mess in `docs.py`, `fields.py`, and `schemas.py`.
We should probably do some brainstorming about it. But that's a thing for a separate PR.

File: frontend/coprs_frontend/coprs/views/apiv3_ns/__init__.py [hunk 1]
```diff
@@ -481,42 +440,55 @@ def call_deprecated_endpoint_method(endpoint_method):
...
+        copr = get_copr(ownername, projectname)
+        if not flask.g.user.can_edit(copr):
+            raise AccessRestricted(
+                "User '{0}' can not see permissions for project '{1}' " \
+                "(missing admin rights)".format(
+                    flask.g.user.name,
+                    '/'.join([ownername, projectname])
+                )
+            )
+
```
[the remark this answers is not in the document]
Comment (nikromen) [reply frontend/coprs_frontend/coprs/views/apiv3_ns/__init__.py:464]: aha, missed this one. nice catch!

[the remark this answers is not in the document]
Comment (nikromen) [reply frontend/coprs_frontend/coprs/views/apiv3_ns/__init__.py:464]: aha, missed this one... nice catch!

**Review by FrostyX — APPROVED** [review]

**Pull request merged**